renderer/index.js

```diff
@@ -48,12 +56,22 @@ function init () {
   })
   document.body.appendChild(vdomLoop.target)
 
+  // Calling update() updates the UI given the current state
+  // For performance, we don't want to do this too often. Hence, events that
+  // affect the UI can be v frequent--like torrent progress updates--call
+  // updateThrottled() instead, which calls update() at most once a second.
   updateThrottled = throttle(update, 1000)
+  setInterval(updateThrottled, 1000)
```

Comment:
Eventually I want to get rid of the `setInterval` since we don't want to do DOM diffing every 1s. That's super expensive, especially when the app is in the background. But yeah this works for now :)